### DISCUSSION TITLE
Installing PHP extensions

### DISCUSSION THREAD

**Discussion by mahemmati** [vitodeploy/vito#60] [Q&A] [answered]
What's the recommended way to install PHP extensions? Should I install them with `sudo apt-get install` via SSH?

**Comment by saeedvaziry** [comment, marked as answer]
This is already on the list #6 

But as a workaround here you can see how we will handle installing the extensions

https://github.com/vitodeploy/vito/blob/main/resources/commands/php/install-php-extension.sh
